app/views/foreman_ansible/job_templates/ansible_-_inventory.erb

```diff
@@ -0,0 +1,30 @@
+<%#
+name: Ansible Inventory
+snippet: false
+template_inputs:
+- name: required
+  required: false
+  input_type: user
+  description: Comma-separated values that should be returned.
+  advanced: false
+  value_type: plain
+
+model: ReportTemplate
+require:
+- plugin: foreman_ansible
+-%>
+<%- load_hosts.each_record do |host| -%>
+<%- inventory_data = {
+      'Name': host.name
+    }
+    required = input('required').split(',').map { |r| r.strip.downcase }
+    inventory_data.merge!('Organization': host.organization) if (['all', 'organization'] & required).any?
+    inventory_data.merge!('Location': host.location) if (['all', 'location'] & required).any?
+    inventory_data.merge!('Hostgroup': host.hostgroup.name) if (['all', 'hostgroup'] & required).any?
```

Comment:
```suggestion
    inventory_data.merge!('Hostgroup': host.hostgroup.name) if (['all', 'hostgroup'] & required).any? && host.hostgroup
```
to avoid nil reference.

Reply:
So based on the new [fields](https://gist.github.com/sjha4/eee846297dcf051e9a4f14198b444309) we want to add, I tried this:
`['architecture_id','architecture_name'].each do |field|
inventory_data.merge!(field=>host[field])`

nor can I do something like `inventory_data.merge!(field=>host.send(field))`

It seems the` Host::Managed::Jail` does not like it.